index.html

```diff
@@ -166,8 +167,36 @@ <h3>Skills</h3>
                 </li>  
             </ul>
           
-        </section>       
+        </section> 
+
     </main> 
- 
+    
+    <footer> 
+        <p class="primary-text">
+            I'm always interested in hearing about new projects,
+             so if you'd like to chat please get in touch.</p>
+
+        <form action="https://formspree.io/f/xpzbzzjw" method="POST">
+
+           <input type="text" id="fname" name="Fullname" placeholder="Full name" maxlength="30" required>
+            
+           <input type="email" id="email" name="email" placeholder="Email Address" required>
+            
+           <textarea id="message" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+            
+           <input type="submit" id="submit" value="Get in touch">
```

Comment:
- [ ] Hi, everything else about your design is really good 👍. In order to have this template match the Figma perfectly please ensure that you implement the interaction state correctly as per the Figma template. 
please implement the `:active` state for this button
## Your Design
![image](https://user-images.githubusercontent.com/18730106/145086115-e7fcb867-958f-4675-851f-80d278e664de.png)
## Figma Design (Interaction states)
![image](https://user-images.githubusercontent.com/18730106/145086211-bdd69816-b435-4083-b4ba-d8e3d2c96344.png)